homeassistant/components/bthome/sensor.py

```diff
@@ -260,6 +263,10 @@
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    # Text (-)
+    (BTHomeExtendedSensorDeviceClass.TEXT, None): SensorEntityDescription(
+        key=f"{BTHomeExtendedSensorDeviceClass.TEXT}",
```

Comment:
Should these be marked as diagnostics... ?

I guess we don't really know how they will be used though?

Reply:
I don't think we should use diagnostics here. It is/can be a normal sensor that is broadcasting some text string. It's up to the user what text string he/she needs for his/her project.